accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+    - `get_Title()`
+    - `MoveBufferArea(Int32, Int32, Int32, Int32, Int32, Int32)`
+    - `MoveBufferArea(Int32, Int32, Int32, Int32, Int32, Int32, Char, ConsoleColor, ConsoleColor)`
+    - `set_BufferHeight(Int32)`
+    - `set_BufferWidth(Int32)`
+    - `set_CursorSize(Int32)`
+    - `set_WindowHeight(Int32)`
+    - `set_WindowLeft(Int32)`
+    - `set_WindowTop(Int32)`
+    - `set_WindowWidth(Int32)`
+    - `SetBufferSize(Int32, Int32)`
+    - `SetWindowPosition(Int32, Int32)`
+    - `SetWindowSize(Int32, Int32)`
+
+### System.Configuration
+
+> **OPEN** Should we mark the entire type?
+
+* **DpapiProtectedConfigurationProvider**
+    - `Decrypt(XmlNode)`
+    - `Encrypt(XmlNode)`
+
+### System.Diagnostics
+
+* **Process**
+    - `GetProcessById(Int32, String)`
+    - `GetProcesses(String)`
+    - `GetProcessesByName(String, String)`
+    - `set_MaxWorkingSet(IntPtr)`
+    - `set_MinWorkingSet(IntPtr)`
+    - `Start(String, String, SecureString, String)`
+    - `Start(String, String, String, SecureString, String)`
+* **ProcessStartInfo**
+    - `get_Domain()`
+    - `get_LoadUserProfile()`
+    - `get_Password()`
+    - `get_PasswordInClearText()`
+    - `set_Domain(String)`
+    - `set_LoadUserProfile(Boolean)`
+    - `set_Password(SecureString)`
+    - `set_PasswordInClearText(String)`
+* **ProcessThread**
+    - `set_PriorityLevel(ThreadPriorityLevel)`
+    - `set_ProcessorAffinity(IntPtr)`
+
+### System.Drawing
+
+> **OPEN** Should we mark the entire type?
+
+* **Graphics**
+    - `CopyFromScreen(Int32, Int32, Int32, Int32, Size)`
+    - `CopyFromScreen(Int32, Int32, Int32, Int32, Size, CopyPixelOperation)`
+    - `CopyFromScreen(Point, Point, Size)`
+    - `CopyFromScreen(Point, Point, Size, CopyPixelOperation)`
+
+### System.IO.MemoryMappedFiles
+
+* **MemoryMappedFile**
+    - `CreateFromFile(FileStream, String, Int64, MemoryMappedFileAccess, HandleInheritability, Boolean)`
+    - `CreateFromFile(String)`
+    - `CreateFromFile(String, FileMode)`
+    - `CreateFromFile(String, FileMode, String)`
+    - `CreateFromFile(String, FileMode, String, Int64)`
+    - `CreateFromFile(String, FileMode, String, Int64, MemoryMappedFileAccess)`
+    - `CreateNew(String, Int64)`
+    - `CreateNew(String, Int64, MemoryMappedFileAccess)`
+    - `CreateNew(String, Int64, MemoryMappedFileAccess, MemoryMappedFileOptions, HandleInheritability)`
+    - `CreateOrOpen(String, Int64)`
+    - `CreateOrOpen(String, Int64, MemoryMappedFileAccess)`
+    - `CreateOrOpen(String, Int64, MemoryMappedFileAccess, MemoryMappedFileOptions, HandleInheritability)`
+    - `OpenExisting(String)`
+    - `OpenExisting(String, MemoryMappedFileRights)`
+    - `OpenExisting(String, MemoryMappedFileRights, HandleInheritability)`
+
+### System.IO.Pipes
+
+* **NamedPipeClientStream**
+    - `get_NumberOfServerInstances()`
+* **NamedPipeServerStream**
+    - `RunAsClient(PipeStreamImpersonationWorker)`
+* **PipeStream**
+    - `set_ReadMode(PipeTransmissionMode)`
```

Comment:
This works on unix as long as it's set to PipeTransmissionMode.Byte.  We shouldn't just blanketly say it's not supported on unix; a more nuanced mechanism is needed.